docs/src/main/paradox/compatibility-guidelines.md

```diff
@@ -61,6 +61,8 @@ Java
 Scala
 :   ```scala
     akka.http.scaladsl.ClientTransport
+    akka.http.scaladsl.ConnectionContext#httpsClient
+    akka.http.scaladsl.ConnectionContext#httpsServer
```

Comment:
Are the javadsl equivalents missing on the list or is it some scaladsl-specific API change?

Reply:
Now added, thanks.